docs/management/api/event-filters-api.asciidoc

```diff
@@ -0,0 +1,596 @@
+:endpoint-artifact-list-id:     endpoint_events
+:endpoint-artifact-name:        Event Filters
+:endpoint-artifact-name-single: Event Filter
+:endpoint-artifact-name-sentence-start: Event filters
+:endpoint-artifact-name-single-sentence-start: Event filter
+:endpoint-artifact-api-doc-name: event-filter
+
+[[event-filters-api]]
+=== {endpoint-artifact-name}
```

Comment:
Combined with my edit above, I think this attribute needs to be replaced, so that the page title has the right capitalization:
```suggestion
=== {endpoint-artifact-name-sentence-start}
```